docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,294 @@
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/my_classification_task/schema.json
+
+    {
+        "categories": [
+            // list of categories
+        ]
+    }
+
+For example, let's say we are working with a classification model predicting the weather on an image.
+The three classes are sunny, clouded, and rainy.
+
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/classification_weather/schema.json
+
+    {
+        "categories": [
+            {
+                "id": 0,
+                "name": "sunny",
+            },
+            {
+                "id": 1,
+                "name": "clouded",
+            },
+            {
+                "id": 2,
+                "name": "rainy",
+            }
+        ]
+    }
+
+
+
+Prediction Files
+^^^^^^^^^^^^^^^^
+Lightly requires a single prediction file per image. The file should be a .json
+following the format defined under :ref:`Prediction Format` and stored in the subdirectory 
+`.lightly/predictions/${TASK_NAME}` in the storage bucket the dataset was configured with.
+In order to make sure Lightly can match the predictions to the correct source image, 
+it's necessary to follow the naming convention:
+
+.. code-block:: bash
+
+    # filename of the prediction for image FILENAME.EXT
+    .lightly/predictions/${TASK_NAME}/${FILENAME}.json
+
+    # example: my_image.png, classification
+    .lightly/predictions/my_classification_task/my_image.json
+
+    # example: my_subdir/my_image.png, classification
+    .lightly/predictions/my_classification_task/my_subdir/my_image.json
+
+
+Prediction Files for Videos
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When working with videos, Lightly requires a prediction file per frame. Lightly
+uses a naming convention to identify frames: The filename of a frame consists of
+the video filename, the video format, and the frame number (padded with 8 zeros)
+separated by hyphens:
+
+.. code-block:: bash
+
+    # filename of the predictions of the Xth frame of video FILENAME.EXT
+    .lightly/predictions/${TASK_NAME}/${FILENAME}-${X:08d}-${EXT}.json
+
+    # example: my_video.mp4, frame 99
+    .lightly/predictions/my_classification_task/my_video-00000099-mp4.json
+
+    # example: my_subdir/my_video.mp4, frame 99
+    .lightly/predictions/my_classification_task/my_subdir/my_video-00000099-mp4.json
+
+
+Prediction Format
+^^^^^^^^^^^^^^^^^
+Predictions for an image must follow the following format:
+
+.. code-block:: javascript
+
+    {
+        "file_name"           : str,
+        "predictions":        : [],
+    }
+
+Here, `file_name` serves as a unique identifier to retrieve the image for which
+the predictions are made and predictions is a list of `Prediction Singletons` for the corresponding task.
+
+For example:
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/classification_weather/my_image.json
+
+    {
+        "file_name": "my_image.png"
+        "predictions": [
+            // list of prediction singletons
```

Comment:
Let's make this a real example using the same weather condition classes. Giving a "real" example is super valuable and easier to explain than explaining concepts. And here we have plenty of concepts: `tasks`, `prediction schema`, `prediction files`, `prediction singletons`